indy_node/test/write_permission/test_revocation_write_permission.py

```diff
@@ -5,11 +5,21 @@
     CLAIM_DEF_SIGNATURE_TYPE, CLAIM_DEF_TAG
 from indy_common.state.domain import make_state_path_for_claim_def
 from indy_node.test.anon_creds.conftest import claim_def, build_revoc_reg_entry_for_given_revoc_reg_def, \
-    build_revoc_def_by_default
+    build_revoc_def_by_default, build_revoc_def_by_client, build_revoc_def_by_trust_anchor
 from indy_node.test.schema.test_send_get_schema import send_schema_seq_no
+from plenum.common.exceptions import RequestNackedException, RequestRejectedException
 from plenum.test.helper import sdk_sign_request_from_dict, sdk_send_and_check
 
 
+@pytest.fixture(scope='module')
+def tconf_false(tconf):
```

Comment:
`ANYONE_CAN_WRITE` will be removed soon, which means that here we can assume that ANYONE_CAN_WRITE  is always False.
So, I believe the `tconf` below is not needed at all.